docs/how-to-guides/deploy-a-flow/deploy-using-kubernetes.md

```diff
@@ -163,7 +163,7 @@ name: open_ai_connection
 module: promptflow.connections
 api_key: ${env:OPEN_AI_CONNECTION_API_KEY} # env reference
 ```
-You'll need to set up the environment variables in the container to make the connections work.
+You'll need to set up the environment variables in the container to make the connections work. For a full list of environment variables see [Load from environment variables](https://microsoft.github.io/promptflow/how-to-guides/manage-connections.html#load-from-environment-variables).
```

Comment:
This "manage-connections" link is not for deployment, we need a section for this in the deployment doc

Reply:
Where would you propose including the section? Within the same document (`deploy-using-kubernetes.md`) or should I create a new `.md` for configuring the connections at deployment time?

Reply:
@wxpjimmy I added a section within the kubernetes how to doc in [651b94c](https://github.com/microsoft/promptflow/pull/3508/commits/651b94c1153a61c60103d1e3b216d43a9db15b80).